src/components/topRail/TopRail.vue

```diff
@@ -85,6 +85,11 @@ export default {
       this.isOpenPanel = value;
     },
   },
+  mounted() {
```

Comment:
This line is introducing testing code in our code. To avoid this, can you check if it's possible to use a  selector to have access to the toprail

Reply:
I will be checked